node_modules/oae-activity/tests/test-email.js

```diff
@@ -1657,4 +1658,42 @@ describe('Activity Email', function() {
             });
         });
     });
+
+    /**
+     * Regression test for one of our more verbose activities (2 or more items shared with a group).
+     * Here we ensure that there is no line that surpasses a safe SMTP threshold
+     */
+    it('verify email for 2 items being created for a group do not result in lines that are too long', function(callback) {
+        TestsUtil.generateTestUsers(camAdminRestContext, 3, function(err, users, simong, nico) {
+            assert.ok(!err);
+            TestsUtil.generateTestGroups(simong.restContext, 1, function(group) {
+                group = group.group;
+
+                // Add the members to the group
+                var members = {};
+                members[nico.user.id] = 'manager';
+                PrincipalsTestUtil.assertSetGroupMembersSucceeds(simong.restContext, group.id, members, function() {
+
+                    // Collect any email notifications so we can focus on just the one we generate next
+                    EmailTestUtil.collectAndFetchAllEmails(function() {
+
+                        // Create 2 links for the group
+                        RestAPI.Content.createLink(simong.restContext, 'Google', 'Google', 'public', 'http://www.myawesomelinkthatilike.ca', [group.id], [], [], function(err) {
```

Comment:
This PR won't fix that as it is right now. We can't take a "container" approach to that, it would have to be handled in the template generation